src/victory-legend/victory-legend.js

```diff
@@ -0,0 +1,252 @@
+import React, { PropTypes } from "react";
+import { PropTypes as CustomPropTypes, TextSize, Helpers } from "../victory-util/index";
+import { merge, isEmpty, defaults, sumBy, maxBy } from "lodash";
+import VictoryLabel from "../victory-label/victory-label";
+import VictoryContainer from "../victory-container/victory-container";
+import Point from "../victory-primitives/point";
+
+const defaultStyles = {
```

Comment:
@angelanicholas Can this stuff go into a theme instead? Can you add a theme prop so that the legends work with themes?